### PR TITLE
Kibana

### PR DESCRIPTION
**Pull request by AWare**
## Why are you doing this?
So membership-frontend logs end up in Kibana. :+1:

## Changes
* Membership Frontend Logs in Kibana
* Membership Frontend Permissions to Use Kinesis

@johnduffell

### CODE REVIEW

File: frontend/app/loghandling/Logstash.scala [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                        awsCredentialsProvider: AWSCredentialsProvider,
+                        customFields: Map[String, String])
+
+object Logstash extends StrictLogging {
+
+  def customFields(playConfig: Config.type) = Map(
+    "app" -> playConfig.appName,
+    "stage" -> playConfig.stage,
+    "build" -> app.BuildInfo.buildNumber,
+    "revision" -> app.BuildInfo.gitCommitId,
```
Comment (johnduffell) [inline frontend/app/loghandling/Logstash.scala]: great on the fields!

**Comment by johnduffell** [comment]
looks good to me, thanks! as long as you've tested it locally go for it! 👍 🐎

**Comment by johnduffell** [comment]
PS don't forget to cloudform it first

**Comment by AWare** [comment]
cloudformed

**Pull request merged**